VersionControlUI/src/nodes/DiffComparisonPair.cpp

```diff
@@ -123,18 +155,21 @@ QString DiffComparisonPair::computeComponentName()
 	else
 		nodeToComputeComponentType = newVersionNode_;
 
-	auto searchCompositeNode = nodeToComputeComponentType;
+	auto searchCompositeNode = nodeToComputeComponentType->parent();
 
 	while (searchCompositeNode && !DCast<Model::CompositeNode>(searchCompositeNode))
+	{
+		nodeToComputeComponentType = searchCompositeNode;
 		searchCompositeNode = searchCompositeNode->parent();
+	}
 
 	if (searchCompositeNode)
 	{
 		auto compositeNode = DCast<Model::CompositeNode>(searchCompositeNode);
 		if (compositeNode != nodeToComputeComponentType)
 		{
 			auto index = compositeNode->indexOf(nodeToComputeComponentType);
-			componentName = compositeNode->meta().attribute(index).name();
+			componentName = " > " + compositeNode->meta().attribute(index).name();
```

Comment:
I'm a bit confused. Now that we have the bread crumbs, do we still need this component name? Maybe I just forgot what it is.